src/iccpd/src/mlacp_sync_update.c

```diff
@@ -347,11 +530,22 @@ int mlacp_fsm_update_mac_entry_from_peer( struct CSM* csm, struct mLACPMACData *
                 if (csm->peer_link_if && csm->peer_link_if->state == PORT_STATE_UP)
                     add_mac_to_chip(mac_msg, mac_msg->fdb_type);
```

Comment:
@Praveen-Brcm 
The mac address is added only when peer-link is configured and the link is up.